### PR TITLE
add tests for comment stream for non-admins

### PR DESCRIPTION
**Pull request by riley**
## What does this PR do?

Exactly what it says on the tin.

## How do I test this PR?

- `npm test`

### CODE REVIEW

File: tests/routes/api/comments/index.js [hunk 1]
```diff
@@ -85,6 +85,29 @@ describe('/api/v1/comments', () => {
       ]);
     });
 
+    it('should return only the owner’s comments if the user is not an admin', () => {
+      return chai.request(app)
+        .get('/api/v1/comments?user_id=456')
+        .set(passport.inject({id: '456', roles: []}))
```
Comment (gabelula) [inline tests/routes/api/comments/index.js:91]: You don't have to add the roles field if it is empty.

File: tests/routes/api/comments/index.js [hunk 2]
```diff
@@ -85,6 +85,29 @@ describe('/api/v1/comments', () => {
...
+        .then(res => {
+          expect(res).to.have.status(200);
+          expect(res.body.comments).to.have.length(2);
+          expect(res.body.comments[0]).to.have.property('id', comments[1].id);
```
Comment (gabelula) [inline tests/routes/api/comments/index.js]: Shouldn't be better to just check the user_id property and see that is the user 456 ?

**Review by gabelula — APPROVED** [review]
It looks good.

File: tests/routes/api/comments/index.js [hunk 3]
```diff
@@ -92,7 +92,7 @@ describe('/api/v1/comments', () => {
         .then(res => {
           expect(res).to.have.status(200);
           expect(res.body.comments).to.have.length(2);
-          expect(res.body.comments[0]).to.have.property('id', comments[1].id);
+          expect(res.body.comments[0]).to.have.property('author_id', '456');
```
Comment (gabelula) [inline tests/routes/api/comments/index.js]: Can we add 

expect(res.body.comments[1]).to.have.property('author_id', '456');

?

Comment (riley) [reply tests/routes/api/comments/index.js]: yeah, that makes more sense

**Pull request merged**